scripts/build/appveyor/configure_appveyor_VS2017.bat

```diff
@@ -2,6 +2,7 @@ cmake .. -G "Visual Studio 15 2017 Win64"
 -DCMAKE_CXX_FLAGS="/D KRATOS_DISABLE_AMGCL" 								    ^
 -DCMAKE_C_FLAGS="/D KRATOS_DISABLE_AMGCL" 									    ^
 -DBOOST_ROOT="C:\Libraries\boost_1_65_1"                                        ^
+-DPYTHON_EXECUTABLE="C:\Python27-x64\python.exe"							        ^
 -DPYTHON_INCLUDE_PATH="C:\Python27-x64\include"							        ^
```

Comment:
i'll accept the PR, but if you pass the python exe you don't need the INCLUDE and LIB path (i think)